generators/ci-cd/templates/_.gitlab-ci.yml

```diff
@@ -24,39 +22,59 @@ stages:
     - release
     - deploy
 
-<%_ if (buildTool == 'gradle') { _%>
+<%_ if (buildTool === 'gradle') { _%>
 before_script:
     - export GRADLE_USER_HOME=`pwd`/.gradle
+    - chmod +x gradlew
```

Comment:
for `jenkins` it makes sense. But even for `gitlab` do we need to have ? 
But it is okay to be like this 👍